multimodal/tests/unittests/others/test_deployment.py

```diff
@@ -65,9 +68,7 @@ def test_onnx_export(checkpoint_name):
     "checkpoint_name,num_gpus",
     [
         ("swin_tiny_patch4_window7_224", -1),
-        ("vit_tiny_patch16_224", -1),
         ("resnet18", 0),
-        ("legacy_seresnet18", -1),
     ],
 )
 def test_onnx_export_image_cls(checkpoint_name, num_gpus):
```

Comment:
How about using `test_onnx_export_timm_image` to align with the above `test_onnx_export_hf_text `?

Reply:
Renamed accordingly.